### PR TITLE
BREAKING CHANGE: updates footer closed props

### PR DESCRIPTION
**Pull request by hyper0009**
* Updates footer props to include `MarketingImagePath` so we can pass down imgs from assets
* Updates `index.stories.tsx` to pass down the props for the storybook demo

### CODE REVIEW

File: packages/footer-dock-closed/index.stories.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import * as React from "react";
 import { storiesOf } from "@storybook/react";
 import { withKnobs, boolean, text } from "@storybook/addon-knobs";
-
 /* Themed Component */
 import { ThemeProvider } from "styled-components";
 import { LightComponentTheme } from "@times-paywall-components/style-guide";
 
 import FooterClosed from "./src/index";
+const TheTimesDeviceSpread = require("./assets/the_times_device_spread.png") as string;
```
Comment (chrishutchinson) [inline packages/footer-dock-closed/index.stories.tsx]: We should be able to do a regular `import` for this. Does TypeScript complain?

Comment (hyper0009) [reply packages/footer-dock-closed/index.stories.tsx]: yep, `TypeScript` complains even if I add it to the type declaration file

Comment (chrishutchinson) [reply packages/footer-dock-closed/index.stories.tsx]: Nightmare! Well its just a story so pretty is bearable 👍

Comment (hyper0009) [reply packages/footer-dock-closed/index.stories.tsx]: well, it seems fine now that I changed it again. will push up the changes

**Pull request merged**